airbyte-cdk/python/airbyte_cdk/sources/streams/http/http.py

```diff
@@ -103,6 +103,13 @@ def max_retries(self) -> Union[int, None]:
         """
         return 5
 
+    @property
+    def max_time(self) -> Union[int, None]:
+        """
+        Override if needed. Specifies maximum total waiting time (in seconds) for backoff policy. Return None for no limit.
+        """
+        return None
```

Comment:
My plan is to set this value to '10 minutes' for certified connectors, while we can use higher values for standard ones. 
I suggest setting it to 20 minutes to start.

Reply:
why not set it to 10 minutes and customize it for connectors that need more time?

1. This would remove one manual step from the certification process
2. Being stuck in a retry loop for 15 minutes is indicative of a problem. It might be fine for non-certified connectors, but explicitly increasing the timeout would serve as documentation

Reply:
Yes, we could use this approach